packages/design-system/adr/process-ds-transformation.md

```diff
@@ -0,0 +1,87 @@
+# Transform Design system process
+
+## Current status of DS?
+
+- A dedicated team → 2 DS dev + 1 DS designer
+- In _collaboration_ with products designers/content designers
+- _Lack of collaboration_ from product developers
+- _Low usage_ in products
+- How can we move from the DS team to something?
+
+## Org proposals
+
+- Have a composite team from the different domains (1 or 2 devs per domain) and Designers => `Design System Champions 🎖`
```

Comment:
We've been pushing for exactly this over the past two years, I do hope you're more successful than we were.

Reply:
what difficulties did you face @Benrajalu ? Maybe it could help...